ios/FluentUI/People Picker/ContactCollectionView.swift

```diff
@@ -111,7 +111,7 @@ open class ContactCollectionView: UICollectionView {
     }
 
     @objc private func updateHeightConstraint() {
-        let height = UIApplication.shared.preferredContentSizeCategory.contactHeight(size: size.contactViewSize) + 2 * Constants.verticalInset
+        let height = self.traitCollection.preferredContentSizeCategory.contactHeight(size: size.contactViewSize) + 2 * Constants.verticalInset
```

Comment:
nit: can you remove the redundant "self." from these calls?

Reply:
Ok.